hmda/src/main/scala/hmda/validation/dsl/PredicateCommon.scala

```diff
@@ -38,6 +38,8 @@ object PredicateCommon {
     Try(lessThanOrEqual(BigDecimal(upper)).check(BigDecimal(x))).getOrElse(false) && Try(greaterThanOrEqual(BigDecimal(lower)).check(BigDecimal(x))).getOrElse(false)
   }
 
+  def containedIn[A](domain: Set[A]): Predicate[A] = domain.contains(_: A)
+
   def containedIn[A](domain: Seq[A]): Predicate[A] = domain.contains(_: A)
```

Comment:
are these 2 duplicated?

Reply:
one uses `Seq` other uses `Set`. I added a commit to change the names

Reply:
ah~ yeah, I see it now, I'm blind...